src/commands.ts

```diff
@@ -14,7 +14,8 @@ async function install ( file: vscode.Uri ) {
   /* TERMINAL */
 
   const term = vscode.window.createTerminal ( 'VSIX' ),
-        command = Utils.isInsiders () ? 'code-insiders' : 'code';
+        command = Utils.isInsiders () ? 'code-insiders' :
```

Comment:
can you write this in one line? Just wrap the second ternary operator in parenthesis for clarity.